swim/update_rollup_test.go

```diff
@@ -101,7 +103,7 @@ func (s *RollupTestSuite) TestFlushesAfterInterval() {
 	s.NotNil(s.r.FlushTimer(), "expected timer to be set")
 	s.Len(s.r.Buffer(), 2, "expected two updates to be in buffer")
 
-	time.Sleep(2 * time.Millisecond)
+	s.node.clock.(*clock.Mock).Add(2 * time.Millisecond)
```

Comment:
I think its a bit cleaner if we didn't need the cast
s.clock.Add(2*..)
This means need to add the field to the testSuite. 
Not a blocker though